openqa-investigate

```diff
@@ -374,7 +374,17 @@ investigate() {
     comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id") || rc=$?
     [[ $rc != 255 ]] && return $rc
 
-    out=$(trigger_jobs "$id" "${@:2}") || return $?
+    echo "------- comment_id: $comment_id"
+    out=""
+    if ! out=$(trigger_jobs "$id" "${@:2}"); then
+        echo "------- out: $out"
+        client-delete-job-comment "$first_cluster_job_id" "$comment_id"
+        echoerr "Unable to trigger investigation jobs for job $id"
+        return 1
+    fi
+
+    echo "------- out: $out"
```

Comment:
I assume this is manual debugging left-over?

Reply:
Yep, I forgot to mark the pr as draft, I'll remove those before presenting the final version.